src/csharp/Grpc.Tools.MSBuildIntegration/Grpc.Tools.MSBuildIntegration.targets

```diff
@@ -0,0 +1,27 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">

```

Comment:
I think it should, thanks to this
```
    <Touch Files="@(Proto->'$(IntermediateOutputPath)%(FileName).cs')" ForceTouch="true">
        <Output TaskParameter="TouchedFiles" ItemName="Compile" />
    </Touch>
```
Here the files are added to the Compile collection (presumbaly only if they exist, the [documentation](https://msdn.microsoft.com/en-us/library/37fwbyt5.aspx) does not state the default for AlwaysCreate though.)